scripts/deploy/main.bicep

```diff
@@ -248,10 +248,6 @@ resource appServiceWebConfig 'Microsoft.Web/sites/config@2022-09-01' = {
         name: 'MemoryStore:Qdrant:Port'
         value: '443'
       }
-      {
-        name: 'MemoryStore:AzureCognitiveSearch:UseVectorSearch'
-        value: 'true'
-      }
```

Comment:
Why is this being deleted?

Reply:
This has been removed from code ages ago but was forgotten in the bicep file